addOns/client/src/main/java/org/zaproxy/addon/client/ExtensionClientIntegration.java

```diff
@@ -542,13 +560,29 @@ public void addReportedObject(ReportedObject obj) {
                 .forEach(
                         s -> {
                             try {
+                                incScanCount();
```

Comment:
I'd have expected this outside the lambda, should not need to count per scan rule executed but for each reported object.